apps/passport-server/src/util/telegramHelpers.ts

```diff
@@ -34,6 +38,14 @@ export const getSessionKey = (ctx: Context): string | undefined => {
     : `${ctx.from.id}/${ctx.chat.id}`;
 };
 
+export const isDirectMessage = (ctx: Context): boolean => {
+  return !!(ctx.chat?.type && ctx.chat?.type === "private");
```

Comment:
note: it could be nice to have this type as an enum, i.e.

```typescript
enum TelegramChatType {
  PRIVATE = "private",
  SUPERGROUP = "supergroup"
}

Reply:
Typescript autocompletes these values from the bot Context object, so I think its ok for now.